open_academy/models/partner.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+    instructor = fields.Boolean()
+    teacher_type = fields.Selection(
+        selection=[('level1', 'Level 1'), ('level2', 'Level 2')],
+    )
+    session_ids = fields.Many2many('session', string='Relation')
```

Comment:
remove the string it is not needed and is not a good name for a relational field

if you read it without context it only says relation, but relation to what?

```suggestion
    session_ids = fields.Many2many('session')
```